src/util/mercury_request.h

```diff
@@ -137,6 +138,17 @@ hg_request_waitall(int count, hg_request_t *request[],  unsigned int timeout,
         unsigned int *flag);
 
 /**
+ * A convenience callback function to pass to HG_Forward when using
+ * hg_request. This function simply interprets the argument passed into
+ * HG_Forward as an hg_request_t and calls hg_request_complete on it
+ *
+ * \return HG_SUCCESS if hg_request_complete returns HG_UTIL_SUCCESS,
+ *         HG_OTHER_ERROR otherwise
+ */
+HG_UTIL_EXPORT hg_return_t
+hg_request_forward_cb(const struct hg_cb_info *cb_info);
```

Comment:
Sure! How about `hg_request_complete_cb` - a callback that calls hg_request_complete?

BTW, nice that the callback types have been merged.